dto/dto_bingx_order.py

```diff
@@ -1,8 +1,9 @@
 import json
```

Comment:
Apply this change to trailling stop loss task instead of here

Reply:
Pushed to feature/orderPlacingLogic branch